Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/SurfaceManipulator.cpp

```diff
@@ -78,12 +60,15 @@ namespace AzToolsFramework
     {
         SetSpace(worldFromLocal);
         AttachLeftMouseDownImpl();
+
+        m_rayRequest.m_onlyVisible = true;
```

Comment:
I thought this was self-evident, the surface manipulator only wants to intersect against visible objects/entities/meshes, before this setting was in one of the intersect calls, but as we cache the `m_rayRequest` (as it will store the ignore `EntityId` list) we set it here in the constructor, I can add a comment for extra clarity.